### ISSUE TITLE
Change OSX USE_OMP default

### ISSUE THREAD

**Issue by djhoese** [storpipfugl/pykdtree#37]
Related to #8, I can still get the below error message when installing pykdtree in a non-conda environment:

```
$ pip install pykdtree
Collecting pykdtree
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/aa/29/dcea65f86356bfa032b32cc326db7d84c272c5c8fc5a82f485c247592a75/pykdtree-1.3.1.tar.gz
Requirement already satisfied: numpy in /opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages (from pykdtree)
Installing collected packages: pykdtree
  Running setup.py install for pykdtree ... error
    Complete output from command /opt/local/Library/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python -u -c "import setuptools, tokenize;__file__='/private/var/folders/k6/nsv7qg197bj58rkw2b7mb2ch0000gn/T/pip-build-wW42Xf/pykdtree/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /var/folders/k6/nsv7qg197bj58rkw2b7mb2ch0000gn/T/pip-Zd3T0V-record/install-record.txt --single-version-externally-managed --compile:
    running install
    running build
    running build_py
    creating build
    creating build/lib.macosx-10.13-x86_64-2.7
    creating build/lib.macosx-10.13-x86_64-2.7/pykdtree
    copying pykdtree/test_tree.py -> build/lib.macosx-10.13-x86_64-2.7/pykdtree
    copying pykdtree/__init__.py -> build/lib.macosx-10.13-x86_64-2.7/pykdtree
    running build_ext
    building 'pykdtree.kdtree' extension
    creating build/temp.macosx-10.13-x86_64-2.7
    creating build/temp.macosx-10.13-x86_64-2.7/pykdtree
    /usr/bin/clang -fno-strict-aliasing -fno-common -dynamic -pipe -Os -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I/opt/local/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7 -I/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/numpy/core/include -c pykdtree/kdtree.c -o build/temp.macosx-10.13-x86_64-2.7/pykdtree/kdtree.o -std=c99 -O3 -fopenmp
    clang: error: unsupported option '-fopenmp'
    error: command '/usr/bin/clang' failed with exit status 1
```

However, if I `pip install pykdtree` in a conda environment (which provides its own compiler and openmp) then it builds fine. @storpipfugl would you be open to me changing the default to False if the platform is darwin and the python environment is not from conda?

**Comment by djhoese** [comment]
How to determine if python is in a conda environment: https://stackoverflow.com/a/21318941/433202

**Comment by storpipfugl** [comment]
I think that would appropriate as Apple for some reason does not seem to ever want to build in the upstream clang support (#10 )

**Comment by megretson** [comment]
I am still experiencing the above failure in attempting to pip install pykdtree in a non-conda environment (same error output as @djhoese ). Any support for this issue yet, as this seems to be the only source I can find referencing the problem?

**Comment by djhoese** [comment]
@megretson This was fixed in #43 but I don't think it has been released yet.

@storpipfugl @mraspaud Any chance we can get a patch release?
